backend/business-partner-agent/src/main/java/org/hyperledger/bpa/controller/PartnerController.java

```diff
@@ -234,12 +238,16 @@ public HttpResponse<Void> requestCredential(
     /**
      * List proof exchange records
      *
-     * @param id {@link UUID} the partner id
+     * @param pc              {@link PaginationCommand}
+     * @param id              {@link UUID} the partner id
      * @return HTTP status
      */
-    @Get("/{id}/proof-exchanges")
-    public HttpResponse<List<AriesProofExchange>> getPartnerProofs(@PathVariable UUID id) {
-        return HttpResponse.ok(proofM.listPartnerProofs(id));
+    @Get("/{id}/proof-exchanges{?pc*}")
+    public HttpResponse<Page<AriesProofExchange>> getPartnerProofs(
+      @Valid @Nullable PaginationCommand pc,
+      @Parameter(description = "partner id") @Nullable @QueryValue UUID id) {
```

Comment:
please change this to: `@PathVariable UUID id` see spotbugs

Reply:
done